dotCMS/src/main/java/com/dotcms/rest/api/v1/sites/ruleengine/rules/RuleResource.java

```diff
@@ -213,26 +216,19 @@ private User getUser(final HttpServletRequest request, final HttpServletResponse
         return webResource.init(request, response, true).getUser();
     }
 
-    private List<RestRule> getRulesInternal(User user, Ruleable host) {
-        try {
-            List<RestRule> restRules = new ArrayList<>();
+    private List<RestRule> getRulesInternal(User user, Ruleable host) throws DotSecurityException, DotDataException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093298)